src/desktop/apps/user/components/categories/view.coffee

```diff
@@ -67,7 +67,7 @@ module.exports = class CategoriesView extends QuasiInfiniteView
         view = new FollowButton
           el: @$(".js-entity-follow[data-id='#{category.id}']")
           following: @allFollows
-          modelName: 'category'
+          modelName: 'gene'
```

Comment:
This was a bug, we sort of interchangably call genes categories and vice versa, but in terms of the code (which generates the API route, etc.), its `gene`.